packages/canvas/canvas-sprite/src/CanvasTinter.js

```diff
@@ -274,21 +274,14 @@ const CanvasTinter = {
      * The tinting method that will be used.
      *
      * @memberof PIXI.CanvasTinter
-     * @type {tintMethodFunctionType}
+     * @type {Function}
      */
-    tintMethod: 0,
+    tintMethod: () =>
+    { // jslint-disable no-empty-function
+
+    },
```

Comment:
Why did this change from using the callback typedef?

Reply:
There was a callback? I just removed `tintMethodFunctionType`

Reply:
Yeah the tag is `@callback` which defines a function typedef. I'm asking why you removed that and replaced it with a `Function` type.